tests/integration/minibroker_test.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration_test
+
+import (
+	"fmt"
+
+	apiv1beta1 "github.com/kubernetes-sigs/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"k8s.io/client-go/kubernetes"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/kubernetes-sigs/minibroker/tests/integration/testutil"
+)
+
+const (
+	brokerName = "minibroker"
+)
+
+var (
+	kubeClient kubernetes.Interface
+	sc         *testutil.Svcat
```

Comment:
I initially named it `sc` because I had the `svcat` package being imported. I can see that this is not true anymore and using `svcat` for the global variable is indeed better.

Reply:
Fixed with https://github.com/kubernetes-sigs/minibroker/pull/59/commits/33db236612e065eb02ff2cfc832720f17710ea1e.